src/main/java/org/mifos/connector/channel/zeebe/ZeebeWorkers.java

```diff
@@ -74,7 +74,12 @@ private void workerSendErrorToChannel(){
                                 errorInformation.getString("errorDescription"));
                     }
 
+                    Map<String, Object> existingVariables = job.getVariablesAsMap();
+
+                    existingVariables.put("transferCreateFailed", "false");
```

Comment:
create a String constant for the literal and use

Reply:
DONE